safe/definitionsv4/constants.py

```diff
@@ -33,9 +33,17 @@
     QVariant.ULongLong
 ]
 
+# Extent selector
+HAZARD_EXPOSURE = 'HazardExposure'
+HAZARD_EXPOSURE_VIEW = 'HazardExposureView'
+HAZARD_EXPOSURE_BOOKMARK = 'HazardExposureBookmark'
+HAZARD_EXPOSURE_BOUNDINGBOX = 'HazardExposureBoundingBox'
```

Comment:
@timlinux It's not an enumerated constant. In the settings, the string will be saved.